### DISCUSSION TITLE
Document use of "press" and "type" for different key classes

### DISCUSSION THREAD

**Discussion by daobrien** [StyleGuides/WritingStyleGuide#435] [Ideas]
Brainstorming for now.
Sometimes we see "Press `kbd:[Enter]`", sometimes "Press `q` to quit", sometimes "Type `:q`", and various other combinations. It would be nice to have conventions for press/type/enter and also for the markup that's used.

E.g.
1. "You can get more information about the tasks that were run by the first play by pressing `0`, or by typing `:0` in `ansible-navigator`:"

Why use "pressing" for the first and "typing" for the second? Not saying it's wrong; just asking.

Some more things:
- Press `kbd:[Enter]` to engage warp drive.
- Press `kbd:[Ctrl+Alt+Del]` to eject the warp core.
- Enter `warp 8` in the `btn:[Max Warp]` field.
- Type `:q` to retract the assimilation tubules.

Here's another example:
2. "In the preceding example, if you press `2` or enter `:2`, then details of the "Ensure HAProxy is started and enabled" task are displayed, and you can use the arrow keys to scroll through the results:"

Another example:
3. "If you are running `ansible-navigator` in interactive mode, then you can type the `:help` command to open a help page."
I'd be much more inclined to write "...you can use the `:help` command..." in this case, or "...type `:help` to open a help page."

@mweetman-redhat @StyleGuides/style-guide-committers

**Comment by mweetman-redhat** [comment]
I agree with David's examples... my thinking is that we use `press` for any number of simultaneously used keys, and `type` for keys used in series.
Entering text into a GUI field also seems correct... no arguments from me @daobrien

**Reply by daobrien** [reply]
:+1:

**Comment by julian-cable** [comment]
I largely agree, although I would suggest to limit the use of `press` to keystrokes that do not produce output on a screen, per IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=elements-keyboard-keys
